src/ProxyClient/Invalidation/TagsInterface.php

```diff
@@ -29,13 +29,11 @@
     public function invalidateTags(array $tags);
 
     /**
-     * Get the value for the HTTP tag header.
-     *
-     * This concatenates all tags and ensures correct encoding.
+     * Get escaped tags
      *
      * @param array $tags
      *
-     * @return string
+     * @return array
      */
     public function getTagsHeaderValue(array $tags);
```

Comment:
but if PSR-7 does some escaping, we don't want to escape things it would escape ourselves, or we risk double escaping.